charts/ctlog/templates/_helpers.tpl

```diff
@@ -100,8 +100,16 @@ Selector labels
 {{- define "ctlog.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "ctlog.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/component: {{ ctlog.name | quote }}
 {{- end }}
 
+{{/*
+Match labels
+*/}}
+{{- define "ctlog.matchLabels" -}}
```

Comment:
this isnt used any where

to use it, you need to add `{{- include "ctlog.matchLabels" . | nindent 4 }}` or something similar in the yaml under templates/

Reply:
Ok, done. I made it so it matches what trillian has. so matchLabels in the templates.